### DISCUSSION TITLE
Aiuto con aggiornamenti registro

### DISCUSSION THREAD

**Discussion by lucacaputo09** [iisgiua/giuaschool#694] [Richieste di aiuto]
Salve, quando vado nella sezione aggiornamenti del registro mi appare questa schermata. Potete dirmi come posso procedere all'attivazione degli aggiornamenti?
![Screenshot_20240427_100712](https://github.com/iisgiua/giuaschool/assets/165372166/69005422-a15d-477b-8e0f-905a1a75a8a8)

**Comment by trinko** [comment]
Ti prego di scrivere solo una segnalazione di errore.
Non andare a duplicare la richiesta in più parti.
Chiudo la discussione e rispondo nelle segnalazioni.
